src/siteconfig/models.py

```diff
@@ -1,18 +1,42 @@
 from django.contrib.auth import get_user_model
 from django.core.cache import cache
 from django.core.exceptions import MultipleObjectsReturned
-from django.db import connection, models
+from django.db import connection, models, transaction
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 from django.shortcuts import get_object_or_404
 from django.templatetags.static import static
+from django.conf import settings
 
 from django_tenants.utils import get_public_schema_name
 from redis import exceptions as redis_exceptions
 
 User = get_user_model()
 
 
+def get_default_deck_owner():
+    """ 
+        This is run once during site initialization
+
+        Also need to initialize owner if none exists yet.
+        (Since initialization.py loads after everything else)
+    """ 
+    qs = User.objects.filter(username=settings.TENANT_DEFAULT_OWNER_USERNAME)
+    if qs.exists():
+        return qs.first().pk
+
+    # from testing this should only run the first 3 times?
+    # Duplicates should be prevented
+    # https://stackoverflow.com/questions/21458387/transactionmanagementerror-you-cant-execute-queries-until-the-end-of-the-atom
```

Comment:
Love link to references/docs/etc in the code, thank you.

Reply:
done